rebasehelper/specfile.py

```diff
@@ -410,9 +389,13 @@ def write_updated_patches(self, patches, disable_inapplicable):
                     patch_inapplicable = [x for x in patches['inapplicable'] if patch_name in x]
                 else:
                     patch_inapplicable = None
-                if check_not_applied or patch_removed:
+                if check_not_applied:
                     self.spec_content[index] = '#{0} {1}\n'.format(' '.join(fields[:-1]),
                                                                    os.path.basename(patch_name))
```

Comment:
I'm not sure if it is necessary to keep unused patches in the spec file. Anyway, it would be nice to preserve whitespaces, so it is obvious from the diff that the line in question was not changed, only commented out.